### PR TITLE
Made all of webapp2 pass lint

### PR DESCRIPTION
**Pull request by mymtwcom**
PR must fix next issue #106

NOTE:

1. If user will try to import deprecated package from `webapp2_extras` as example `sessions_memcache` he will got only warning, and package will be not imported from new dir, earlier he got imported package too from new path `webapp2_extras.appengine.` (This was done cuz flake8 didn't allowed to use `unused imports`. But as I know, deprecated packages will be removed, is it huge trouble?
2. Any suggestions for fixing refactored `lambda assignments` on functions - appreciated.

### CODE REVIEW

File: webapp2_extras/sessions_memcache.py [hunk 1]
```diff
@@ -28,4 +28,3 @@
     'webapp2_extras.sessions_memcache is deprecated. '
     'App Engine-specific modules were moved to webapp2_extras.appengine.'),
     stacklevel=1)
-from webapp2_extras.appengine.sessions_memcache import *
```
Comment (jonparrott) [inline webapp2_extras/sessions_memcache.py]: leave these for now, just [ignore](http://flake8.pycqa.org/en/latest/user/ignoring-errors.html#in-line-ignoring-errors) them.

File: webapp2_extras/sessions_ndb.py [hunk 2]
```diff
@@ -28,4 +28,3 @@
     'webapp2_extras.sessions_ndb is deprecated. '
     'App Engine-specific modules were moved to webapp2_extras.appengine.'),
     stacklevel=1)
-from webapp2_extras.appengine.sessions_ndb import *
```
Comment (jonparrott) [inline webapp2_extras/sessions_ndb.py]: same here.

File: webapp2_extras/users.py [hunk 3]
```diff
@@ -28,4 +28,3 @@
     'webapp2_extras.users is deprecated. '
     'App Engine-specific modules were moved to webapp2_extras.appengine.'),
     stacklevel=1)
-from webapp2_extras.appengine.users import *
```
Comment (jonparrott) [inline webapp2_extras/users.py]: and here.

**Comment by jonparrott** [comment]
Thank you for doing this. It looks mostly good to me- just need to fix one thing.

**Comment by mymtwcom** [comment]
@jonparrott , fixed

**Comment by jonparrott** [comment]
LGTM, merging. Thank you for this!

**Pull request merged**